packages/styles/src/components/validation.scss

```diff
@@ -0,0 +1,68 @@
+@use 'sass:map';
+@use '../tokens/elements';
+@use '../functions/tokens';
+@use './../mixins/icons';
+
+$post-validation-error: #b20000;
+$post-validation-success: #107800;
+$post-device-font-family-default: 'Swiss Post Sans v005';
+$post-device-font-size-9: 14px;
+$post-device-font-weight-regular: 400;
+$post-radio-button-icon-width-inner: 24px;
+$post-radio-button-icon-height-inner: 24px;
+$post-validation-list-gap-inline: 8px;
+
+// tokens.$default-map: elements.$post-radio;
+
+/* form -> has class .validated 
+   input -> .valid or .invalid
+   validation msg has class -> validation-msg
+*/
+
+.validation-msg {
+  display: none;
+}
+
+.validated {
+  input[type='radio'].invalid ~ .validation-msg,
+  input[type='checkbox'].invalid ~ .validation-msg {
+    &:before {
+      height: $post-radio-button-icon-height-inner;
+      width: $post-radio-button-icon-width-inner;
+      display: inline-block;
+      margin-right: $post-validation-list-gap-inline;
```

Comment:
Please use `block`and `inline` properties instead of `top`, `bottom`, `right` and `left`. https://developer.mozilla.org/en-US/docs/Web/CSS/margin-inline-end